examples/byoi/ubuntu-fips/dracut.conf

```diff
@@ -0,0 +1,4 @@
+omit_dracutmodules+=" iscsi iscsiroot "
+add_dracutmodules+=" fips "
+# These libraries/hmacs are needed in order to boot. The file names are in /usr/share/initramfs-tools/hooks/fips-*
+install_items+=" /usr/lib/x86_64-linux-gnu/libgcrypt.so.20 /usr/lib/x86_64-linux-gnu/.libgcrypt.so.20.hmac /usr/lib/x86_64-linux-gnu/libssl.so.1.1 /usr/lib/x86_64-linux-gnu/libcrypto.so.1.1 /usr/lib/x86_64-linux-gnu/.libssl.so.1.1.hmac /usr/lib/x86_64-linux-gnu/.libcrypto.so.1.1.hmac "
```

Comment:
as far as I can see only the hidden hmac for libssl and libcrypto are needed, in case we want to minimize the number of files added here. Its still good to have the full list thought.

Also we could move this to immucore directly, there is a `install_optional_items` that only copies those files if they exist so we could have out of the box fips support in dracut via the immucore config. And grow the list if needed with other files for other distros

Reply:
let's do that, however we have to be smart there and scan for the files. let's keep the example for now, we will work later to make it smaller and easier to digest